setup_state.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Might be useful to add some really quick notes on what this script is for in the README